control_msgs/msg/SingleDOFState.msg

```diff
@@ -0,0 +1,32 @@
+# This message presents current controller state of one degree of freedom.
+
+# DoF name, e.g., joint or Cartesian axis name
+string name
+
+# The set point, that is, desired state.
+float64 set_point
+
+# Current value of the process (ie: latest sensor measurement on the controlled value).
+float64 process_value
+
+# First time-derivative of the process value.
+float64 process_value_dot
```

Comment:
Several other places to change `dot` to `derivative`, if you like that suggestion